music/cmd/sidecar_cmds.go

```
@@ -0,0 +1,279 @@
+			fmt.Printf("New sidecar: %+v\n", sidecar)
+		} else {
+			fmt.Printf("Sidecar already known: %+v\n", sidecar)
+		}
+	},
+}
+
+var sidecarIdentifyCmd = &cobra.Command{
+	Use:   "identify",
+	Short: "Identify multi-signer sidecars for a zone",
+	Run: func(cmd *cobra.Command, args []string) {
+		PrepArgs("zonename")
```

golangci-lint (1.62.2)

Error return value is not checked

(errcheck)